versions.properties

```diff
@@ -9,6 +9,8 @@
 ####
 #### NOTE: Some versions are filtered by the rejectVersionIf predicate. See the settings.gradle.kts file.
 
+version.com.google.gms..google-services=4.4.2
```

Comment:
Extra `.` here?

Reply:
This file is strange - it gets auto generated but I'm still not sure why this was added. Going to test and see if I can still get a build to work without this. The double dots might be necessary unfortunately.